src/pages/uxp/reference-js/Modules/FileSystem.md

```diff
@@ -73,18 +73,18 @@ const text = fs.readFileSync("plugin-data:/textFile.txt", {encoding: "utf-8"});
 Writes data to the path asynchronously, appending if desired.
 The format of the file is controlled via the encoding option, and defaults to a binary format.
 
+**Returns**: `Promise<int>` - the length of contents written to the file  
```

Comment:
So close 😉 . But it should be `Promise<number>`

https://github.com/AdobeDocs/uxp/issues/86